esdm/esdm_es_mgr.c

```diff
@@ -427,21 +427,26 @@ static uint32_t esdm_avail_entropy_thresh(void)
 	 * Apply oversampling during initialization according to SP800-90C as
 	 * we request a larger buffer from the ES.
 	 */
-	if (esdm_sp80090c_compliant() && !esdm_state.all_online_nodes_seeded)
-		ent_thresh += ESDM_SEED_BUFFER_INIT_ADD_BITS;
+	if (esdm_sp80090c_compliant()) {
+		if (!esdm_state.all_online_nodes_seeded) {
+			ent_thresh += ESDM_SEED_BUFFER_INIT_ADD_BITS;
+		} else {
+			ent_thresh += ESDM_OVERSAMPLE_ES_BITS;
+		}
+	}
 
 	return ent_thresh;
 }
 
-bool esdm_fully_seeded(bool fully_seeded, uint32_t collected_entropy,
+bool esdm_fully_seeded(bool was_fully_seeded_once, bool fully_seeded, uint32_t collected_entropy,
```

Comment:
See before: I do not think this is right: when the DRBG becomes unseeded, you need to do the full seeding dance once again.

Reply:
Is changed to reflect information about prediction resistant DRBG.